src/index.ts

```diff
@@ -1,2 +1,7 @@
-export * from './client';
-export * from './types';
+import { OpenSeaStreamClient } from './client';
+import * as OpenSeaStreamTypes from './types';
+
+export default {
```

Comment:
Why are we exporting like this? Default exports are generally bad, and break tree-shaking.

Reply:
My end-goal is to be able to import all of this SDK's items from a singular import `@opensea/stream-js`. Maybe I was testing things wrong before (you taught me about `npm/yarn link` today) but I couldn't do it before. Will look into alternatives

Reply:
@royliu2 thoughts here?

Reply:
yeah i dont feel strongly about having a default export - i was suggesting this as an option if you wanted it. to Matej's point it may be better to leave this out - I imagine types and client would be used somewhat separately so should be imported separately by users.

Reply:
In ESM, you can do this like this:

```js
import * as whatever from `@opensea/stream-js`
```

If that is what you want?

Reply:
ok cool, reverted this change